R/data.R

```diff
@@ -96,7 +96,7 @@
 #' Degree Days
 #'
 #' @description The monthly accumulated degree days °C
-#' @format a 3 dimensional array [31 watersheds, 12 months, 21 years]
+#' @format a 3 dimensional array [31 watersheds, 12 months, 22 years]
```

Comment:
Good catch here. Ideally these would be in a different PR since it is documentation changes and not tests but fine to leave in for now.